app/Place/PlaceControllerProvider.php

```diff
@@ -64,12 +64,10 @@ public function connect(Application $app): ControllerCollection
 
     public function register(Application $app): void
     {
-        $app['place_editing_controller'] = $app->share(
```

Comment:
The `place_editing_controller` is still used for the endpoints that are not yet replaced. The class it uses should now be `OfferRestBaseController`